cypress/e2e/ui/Overview/dashboard.cy.js

```diff
@@ -71,6 +72,9 @@ describe('Overview > Dashboard Tests', () => {
           cy.get('.card-pf').then((cards) => {
             const nums = [...Array(cards.length).keys()];
             nums.forEach((index) => {
+              if (cards[index].firstChild.innerText === newCard) {
+                newId = cards[index].children[1].id.split('_')[1].replace('w', '');
```

Comment:
This is really weird to set this value in one test and then access it in another - I was expecting that in the refresh test we would extract the id before we click refresh

Reply:
Yeah I can fix this. Some other tests do this too, will also fix those ones too.